tiltfiles/k8s/gql-engine/templates/gql-engine-depl.yaml

```diff
@@ -36,6 +36,7 @@ spec:
             - --name={{ .Values.env.DB_SCHEMA }}
             - --indexer={{ .Values.indexer_endpoint }}
             - --port={{ .Values.port }}
+            - --subscription
```

Comment:
For future reference, this is the sort of thing I would suggest putting into a separate PR. :wink: